polaris/dataset/_dataset.py

```diff
@@ -222,7 +222,7 @@ def zarr_root(self):
             if saved_on_hub:
                 self._zarr_root = self.client.open_zarr_file(self.owner, self.name, self.zarr_root_path, "r+")
             else:
-                self._zarr_root = zarr.open(self.zarr_root_path, "r+")
+                self._zarr_root = zarr.open_consolidated(self.zarr_root_path, "r+")
```

Comment:
It would be great if we can somehow check whether an archive has been consolidated or not, so we can throw more informative errors. Even for people familiar with Zarr, this might not be intuitive. 

We could either: 
1. Use `try-except` and raise a custom error (e.g. `PolarisZarrError`)
2. Check if the `.zmetadata` file exists before opening the archive. 

I think I would prefer the latter option. What do you think?

Reply:
Yes, I don't think we should throw an error if for some reason it is there but not consolidated so I agree with the second option as well.